wqflask/wqflask/markdown_routes.py

```diff
@@ -9,21 +9,22 @@
 from flask import render_template
 
 glossary_blueprint = Blueprint('glossary_blueprint', __name__)
-
+references_blueprint = Blueprint("references_blueprint", __name__)
+environments_blueprint = Blueprint("environments_blueprint", __name__)
+links_blueprint = Blueprint("links_blueprint", __name__)
+policies_blueprint = Blueprint("policies_blueprint", __name__)
+facilities_blueprint = Blueprint("facilities_blueprint", __name__)
 
 def render_markdown(file_name):
-    """Try to fetch the file name from Github and if that fails, try to
-look for it inside the file system
-
-    """
```

Comment:
**question:** Why'd you remove the doc-strings?

Reply:
oops! that was unintended